modules/tinymce/src/core/main/ts/api/AddOnManager.ts

```diff
@@ -131,9 +131,9 @@ const AddOnManager = <T>(): AddOnManager<T> => {
     return undefined;
   };
 
-  const loadLanguagePack = (name: string, languages: string): void => {
+  const loadLanguagePack = (name: string, languages: string = ''): void => {
     const language = I18n.getCode();
-    const wrappedLanguages = ',' + (languages || '') + ',';
+    const wrappedLanguages = ',' + languages + ',';
 
     if (!language || languages && wrappedLanguages.indexOf(',' + language + ',') === -1) {
```

Comment:
This ended up breaking the tests since if languages is falsy it needed to still attempt to load the language script. Looking into this more I believe the intention was to have `languages` be optional, so I've reverted this and updated it to reflect that.